docker/Dockerfile

```diff
@@ -97,26 +107,24 @@ RUN apt update && apt install -y --no-install-recommends \
     apt-get clean && apt-get autoclean
 
 ENV INVOKEAI_SRC=/opt/invokeai
-ENV VIRTUAL_ENV=/opt/venv
 ENV PYTHON_VERSION=3.11
 ENV INVOKEAI_ROOT=/invokeai
 ENV INVOKEAI_HOST=0.0.0.0
 ENV INVOKEAI_PORT=9090
-ENV PATH="$VIRTUAL_ENV/bin:$INVOKEAI_SRC:$PATH"
+ENV PATH="$INVOKEAI_SRC/.venv/bin:$INVOKEAI_SRC:$PATH"
```

Comment:
Keeping the `.venv` out of the source dir enables using this image for development on non-Linux systems. E.g. someone on a Mac can mount their INVOKEAI_SRC into the image while avoiding compatibility issues with their local macOS-specific .venv. We should keep it as-is

Reply:
alright, I believe I've resolved this so it's back to using /opt/venv